app/views/search/index.html.erb

```diff
@@ -1,19 +1,54 @@
-<% @obituaries_api.each do |obit| %>
-  <section class="obituary_by_name">
-    <div class="headline">
-      <%= obit.headline %>
-    </div>
-    <div class="abstract">
-      <%= obit.abstract %>
-    </div>
-    <div class="web_url">
-      <%= obit.web_url %>
-    </div>
-    <div class="pub_date">
-      <%= obit.pub_date %>
-    </div>
-    <div class="word_count">
-      <%= obit.word_count %>
-    </div>
-  </section>
-<% end %>
+<div class="container">
+  <h1 class="my-4">Search Results (Community)</h1>
+    <table class="table table-sm table-bordered table-striped">
+      <thead class='thead-dark'>
+        <tr>
+          <th scope="col">Image</th>
+          <th scope="col">Name</th>
+          <th scope="col">Date</th>
+          <th scope="col">Location</th>
+          <th scope="col">Age</th>
+          <th scope="col">Source</th>
+        </tr>
+      </thead>
+      <tbody style= "background-color:white;">
+        <% @obituaries.each do |obit| %>
+          <tr id="obit-<%= obit.id %>">
+            <td><%= link_to image_tag(obit.image_url, size: "50x50"), "/obituaries/#{obit.id}" %></td>
```

Comment:
Depending on the image size we get back from NYT we can adjust this size to match